ModularTegustation/tegu_items/enkephalin_rush/vending.dm

```diff
@@ -32,3 +25,67 @@
 
 	default_price = 100
 	input_display_header = "Mining Equipment"
+	var/list/company_products = list(//Wing-specifc pickaxes, flares, mostly flavor stuff.
+		/obj/item/stat_equalizer/mining = 100,
+		/obj/item/pickaxe/lcorp = 100,
+		)
+
+/obj/machinery/vending/lobotomy_mining/Initialize()
+	products = (company_products + products)//we want the unique items at the top of the list
+	..()
```

Comment:
ALWAYS do `. = ..()` or `return ..()` in initialize, otherwise it'll error out in the init logs.
Also initialize should always have `mapload` as a variable
```suggestion
/obj/machinery/vending/lobotomy_mining/Initialize(mapload)
	products = (company_products + products)//we want the unique items at the top of the list
	return ..()
```